docs/search-best-practice.md

```diff
@@ -0,0 +1,68 @@
+# Best Practices for Documentation Search
+Documentation (docs) search is an important part of developer experience. It makes life so easy for developers by enabling them narrow down to sections they’re looking for in the documentation. Choosing the right tool for docs search can be a hassle. 
+
+Here at CNCF,we’ve explored different search tools and we’ve decided to narrow our options to this 3:
+
+1. Lunr
+1. Google search
+1. Docsearch by Algolia
```

Comment:
I wonder if it would be better to use a bulleted list here -- we can imply preference via order without explicitly ranking them. We should also link to the sections below, may make this page more manageable as it grows.

```suggestion
* [Lunr](#lunr)
* [Google search](#programmable-search-engine-by-google)
* [Docsearch by Algolia](#docsearch-by-algolia)
```